test/object-store/migrations.cpp

```diff
@@ -2399,9 +2578,10 @@ TEST_CASE("migration: Additive") {
         realm2->commit_transaction();
 
         REQUIRE_NOTHROW(realm->refresh());
-        REQUIRE(realm->schema() == schema);
+        REQUIRE(realm->schema() != schema);
```

Comment:
@tgoyne I had to change a bunch of tests you pushed, because some properties are now appended if realm is opened in additive mode.